cookies/views/giles.py

```diff
@@ -14,9 +14,12 @@
 from cookies import giles
 from cookies.filters import GilesUploadFilter
 
+import pytz
+import calendar
+from datetime import datetime
 
 def _get_upload_by_id(request, upload_id, *args):
```

Comment:
if that doesn't get an upload object anymore, but a resource, the name should indicate that. It wasn't used anywhere before, right?

Reply:
It wasn't used anywhere. I will create a new function just to be safe.